microk8s-resources/actions/kube-api-audit

```diff
@@ -0,0 +1,69 @@
+apiVersion: audit.k8s.io/v1 # This is required.
```

Comment:
Can we rename this file to have `yaml` extension?

Reply:
yes, included as yaml